mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart

```diff
@@ -131,10 +118,26 @@ class ExifBottomSheet extends HookConsumerWidget {
                 style: TextStyle(fontSize: 11, color: textColor),
               ).tr(),
               buildMap(),
-              if (exifInfo != null &&
-                  exifInfo.city != null &&
-                  exifInfo.state != null)
-                buildLocationText(),
+              RichText(
+                text: TextSpan(
+                  style: TextStyle(
+                    fontSize: 12,
+                    fontWeight: FontWeight.bold,
+                    color: textColor,
+                    fontFamily: 'WorkSans',
+                  ),
+                  children: [
+                    if (exifInfo != null && exifInfo.city != null)
+                      TextSpan(
+                        text: exifInfo.city,
+                      ),
+                    if (exifInfo != null && exifInfo.state != null)

```

Comment:
What if `exifInfo.city == null`? Then you'll output the state with a prepended comma and space.